### PR TITLE
Changed Team Interaction

### PR DESCRIPTION
**Pull request by monotkate**
Team:
- Now has consistent controls with backing
- Changed email button to use ng-click because of formatting issues with ng-href
- On removal of team, it alerts that the back will also be deleted and asks for confirmation
- There is no way to leave the team without deleting your back

Back:
- Added a dialog window that asks for confirmation when deleting your back (so as not to do it accidentally)

Tests:
- Checks for $mdDialog.show()
- Checks that backs delete correctly if user
- Checks that removal from team works correctly

Login
- Added "Login" to make it painfully obvious how you login

I think that's it.  I shouldn't keep adding onto this one.  Probably going to take a break for a bit to restructure my app for a potential January release.  One note for discussion: I think the email team button is kind of weird on the idea page.  I understand if on the team tab, but on the idea page I would expect it to email the idea author, not the whole team.

@YashdalfTheGray @Mctalian @alanlgirard 

Fixes issue #183

### CODE REVIEW

File: src/ideas/ideasView/ideasView.js [hunk 1]
```diff
@@ -593,6 +688,17 @@ angular.module('flintAndSteel')
...
+
+            // Open up an email to team members
+            ctrl.parseTeamEmail = function parseTeamEmail() {
+                var emailString = "mailto:";
+                $scope.idea.team.forEach(function(teamElement) {
+                    if (teamElement.member.mail !== 'undefined') {
+                        emailString += teamElement.member.mail + ';';
+                    }
+                });
+                $window.location = emailString;
```
Comment (Mctalian) [inline src/ideas/ideasView/ideasView.js]: Ah, clever!

**Comment by Mctalian** [comment]
Approved pending the "Remove user from team when they delete their back" portion of the card.

**Comment by monotkate** [comment]
@Mctalian I added it already.....

**Comment by Mctalian** [comment]
:racehorse: that was fast! I'll check it out

**Comment by monotkate** [comment]
@Mctalian Alright, got it.  Fixed clearing the back too, found a bug there after it's removed.  I missed that part and it worked but I didn't open the console error

**Comment by Mctalian** [comment]
This looks good, thanks for fixing that. We'll have to figure out what to do with that "Owner" tag eventually. I see you are adding it if the author backs, which is good for now.

Approved :+1:

**Comment by monotkate** [comment]
Yeah took me a bit to figure that one out since it wasn't in chip types. Awesome. Just need another approve now!

**Comment by Mctalian** [comment]
@bcbrennecke

**Comment by bcbrennecke** [comment]
I checked this over with Bob, looks great!  Approve!

**Pull request merged**